stargazer/stargazer.py

```diff
@@ -193,6 +197,14 @@ def show_confidence_intervals(self, show):
         assert type(show) == bool, 'Please input True/False'
         self.confidence_intervals = show
 
+    def show_pvalues(self):
+        """Show P-values in parentheses below coefficients"""
+        self.sig_stat = 'cov_pvalues'
+
+    def show_tvalues(self):
+        """Show T-statistics in parentheses below coefficients"""
+        self.sig_stat = 'cov_tvalues'
```

Comment:
This method makes sense... as long as it also reverts back ``show_confidence_intervals``

Reply:
As in add `self.show_convidence_intervals(False)`? That makes sense, seems obvious in hindsight.

Reply:
The ideal behavior would be that ``self.show_*(True)`` shows only what is being asked, while ``self.show_*(False)`` removes what is being asked if it is currently shown, and has no effect otherwise. I *think* what you're suggesting works.